packages/icon/src/Icon.spec.tsx

```diff
@@ -52,6 +52,7 @@ describe('packages/Icon/glyphs/', () => {
         readonly [K in string]: string | SVGNodeObject;
       };
 
+      //@ts-expect-error
```

Comment:
Need to look into this more, but wanted to pass our tests :)

Reply:
what changed that would even affect this file?

Reply:
not sure, but the error seems to be coming from the toJson typing